cmd/tracee-ebpf/capabilities.go

```diff
@@ -73,16 +74,47 @@ func getCapabilitiesRequiredByTraceeEvents(cfg *tracee.Config) []cap.Value {
 	return removeDupCaps(caps)
 }
 
+// Retrieve the value of the kernel parameter perf_event_paranoid
+func getKernelPerfEventParanoidValue() int {
+	// Read the kernel parameter perf_event_paranoid
+	value, err := os.ReadFile("/proc/sys/kernel/perf_event_paranoid")
+	// If file read failed, returns the most restrictive value
+	if err != nil {
+		return 4
+	}
+	// Strip the newline character
+	value = value[:len(value)-1]
+	// Convert the value to an integer
+	intVal, err := strconv.ParseInt(string(value), 0, 16)
+	// If conversion failed, returns the most restrictive value
+	if err != nil {
+		return 4
```

Comment:
I think it would be better to use consts for these values,
it will make it self-explanatory.